internal/command/modify/modify.go

```diff
@@ -0,0 +1,226 @@
+// This file is part of libraries-repository-engine.
+//
+// Copyright 2021 ARDUINO SA (http://www.arduino.cc/)
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published
+// by the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+//
+// You can be released from the requirements of the above licenses by purchasing
+// a commercial license. Buying such a license is mandatory if you want to
+// modify or otherwise use the software for commercial activities involving the
+// Arduino software without disclosing the source code of your own applications.
+// To purchase a commercial license, send an email to license@arduino.cc.
+
+// Package modify implements the `modify` CLI subcommand used by the maintainer for modifications to the library registration data.
+package modify
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/arduino/go-paths-helper"
+	"github.com/arduino/libraries-repository-engine/internal/backup"
+	"github.com/arduino/libraries-repository-engine/internal/configuration"
+	"github.com/arduino/libraries-repository-engine/internal/feedback"
+	"github.com/arduino/libraries-repository-engine/internal/libraries"
+	"github.com/arduino/libraries-repository-engine/internal/libraries/archive"
+	"github.com/arduino/libraries-repository-engine/internal/libraries/db"
+	"github.com/arduino/libraries-repository-engine/internal/libraries/metadata"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+)
+
+var config *configuration.Config
+var libraryName string
+var libraryData *db.Library
+var releasesData []*db.Release
+
+// Run executes the command.
+func Run(command *cobra.Command, cliArguments []string) {
+	var err error
+	config = configuration.ReadConf(command.Flags())
+
+	if len(cliArguments) == 0 {
+		feedback.Error("LIBRARY_NAME argument is required")
+		os.Exit(1)
+	}
+	if len(cliArguments) > 1 {
+		feedback.Error("Multiple LIBRARY_NAME arguments are not supported")
+		os.Exit(1)
+	}
```

Comment:
This could be enforced with the `Args` field of `cobra.Command`: https://github.com/spf13/cobra/blob/master/user_guide.md#positional-and-custom-arguments

Should be enough to add `Args: ExactArgs(1)` in `modifyCmd`.